### ISSUE TITLE
Exclude trailing_comma validation on single-line declarations

### ISSUE THREAD

**Issue by ZevEisenberg** [realm/SwiftLint#910]
The typical arguments for requiring a trailing comma in arrays and dictionaries is so that adding or removing elements doesn't dirty adjacent lines in source control. However, that argument doesn't hold for single-line declarations such as this one I just came across in a project:

```swift
static let componentsToZeroForDay: [Calendar.Component] = [.hour, .minute, .second, .nanosecond]
```

It would be nice for SwiftLint to ignore cases like this. There could potentially be a separate rule that all arrays and dictionaries must be multiline, but I personally would leave that one off.

**Comment by ZevEisenberg** [comment]
The existing warning is also annoying in cases of single-element arrays, like this:

```swift
let components = calendar.dateComponents([Calendar.Component.day], from: from, to: to)
```

By the way, my SwiftLint configuration includes:

```yml
trailing_comma:
  mandatory_comma: true
```

**Comment by jpsim** [comment]
Should be pretty easy to only report errors in single-line arrays and dictionaries.

**Comment by ZevEisenberg** [comment]
_Single_-line? Don't you mean _multi_-line?

**Comment by jpsim** [comment]
Yes, sorry, updated.

**Comment by jpsim** [comment]
Fixed by @marcelofabri in #912.
